integration_tests/test_basic.py

```diff
@@ -217,3 +218,70 @@ def test_statesync(cronos):
     )
 
     print("succesfully syncing")
+
+
+def test_transaction(cronos):
+    w3 = cronos.w3
+
+    # send transaction
+    txhash_1 = send_transaction(
+        w3,
+        {"to": ADDRS["community"], "value": 10000, "gasPrice": w3.eth.gas_price},
+        KEYS["validator"],
+    )["transactionHash"]
+    tx1 = w3.eth.get_transaction(txhash_1)
+    assert tx1["transactionIndex"] == 0
+
+    # out of gas
+    try:
+        send_transaction(
+            w3,
+            {
+                "to": ADDRS["community"],
+                "value": 10000,
+                "gasPrice": w3.eth.gas_price,
+                "gas": 1,
+            },
+            KEYS["validator"],
+        )["transactionHash"]
+    except ValueError as error:
+        assert "out of gas" in str(error)
```

Comment:
https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions
there's `pytest.raises` to expect error.

Reply:
But I need to assert the string.

Reply:
Oh raises also ok to assert inside?

Reply:
I'll check

Reply:
yes, `with pytest.raises(...) as exc_info` and assert the `exc_info`.